### PR TITLE
sys-cluster/teleport: add v2.5.6

### PR DESCRIPTION
**Pull request by gclawes**
Add v2.5.6

### CODE REVIEW

File: sys-cluster/teleport/teleport-2.5.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+DEPEND="
+	app-arch/zip
+	>=dev-lang/go-1.9.2"
+RDEPEND=""
+
+src_prepare() {
+	default
+
+	sed -i -e 's/-j 3/-j 1/g' src/${EGO_PN%/*}/Makefile
```
Comment (mgorny) [inline sys-cluster/teleport/teleport-2.5.6.ebuild]: ||die

Comment (gclawes) [reply sys-cluster/teleport/teleport-2.5.6.ebuild]: Fixed.

**Comment by gclawes** [comment]
See here: https://github.com/gentoo/gentoo/pull/5735

This is necessary for the teleport webserver to work properly, the ebuild stripping process removes the web asssets that are built into the teleport binary.  I have checked with the upstream devs, there isn't an alternative way available as of now to package the web assets so they don't get stripped.

**Comment by mgorny** [comment]
Oookay but the files are stripped nevertheless for some reason. So please fix the build system to not strip them if it does that.

**Comment by gclawes** [comment]
I can disable the `-w` ldflag in the makefile with a sed in `src_prepare()`: https://github.com/gravitational/teleport/blob/v2.5.6/Makefile#L24

Out of curiosity, where did you see that QA Notice?  I couldn't reproduce that warning building locally.

**Comment by gclawes** [comment]
Updated, this will build un-stripped binaries by removing the `-w` and `-s` flags: https://github.com/gclawes/gentoo/blob/281fc5cb98edcebf15571e6a15714ea276e6def4/sys-cluster/teleport/teleport-2.5.6.ebuild#L38

**Comment by mgorny** [comment]
Thanks a lot for your work!

**Pull request closed without merging**